client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -148,22 +149,29 @@ class DomainManagementNavigationEnhanced extends React.Component {
 		const { selectedSite, translate, domain } = this.props;
 
 		const wpcomUrl = withoutHttp( getUnmappedUrl( selectedSite ) );
-		const { pointsToWpcom, isPrimary } = domain;
-
-		if ( pointsToWpcom && isPrimary ) {
-			return translate( 'Destination: primary domain for %(wpcomUrl)s', {
-				args: {
-					wpcomUrl,
-				},
-			} );
-		}
-
-		if ( pointsToWpcom && ! isPrimary ) {
-			return translate( 'Destination: %(wpcomUrl)s', {
-				args: {
-					wpcomUrl,
-				},
-			} );
+		const {
+			isPrimary,
+			isPendingIcannVerification,
+			pendingTransfer,
+			pointsToWpcom,
+			registrationDate,
+		} = domain;
+
+		const activating =
+			isRecentlyRegistered( registrationDate ) && ! pendingTransfer && ! isPendingIcannVerification;
```

Comment:
I think we don't need `isPendingIcannVerification` here, so we can remove it.
```suggestion
			isRecentlyRegistered( registrationDate ) && ! pendingTransfer;
```